mediagrains/numpy/videograin.py

```diff
@@ -245,6 +245,14 @@ def __init__(self, meta: VideoGrainMetadataDict, data: GrainDataParameterType):
         else:
             self.component_data = ComponentDataList([])
 
+    @property
+    def length(self) -> int:
+        return self._data.nbytes
+
+    @length.setter
+    def length(self, L: int) -> None:
+        super(VIDEOGRAIN, type(self)).length.fset(self, L)  # type: ignore
```

Comment:
Took me a few reads to work out exactly how this worked. An explanatory comment might be useful. In particular `super(VIDEOGRAIN, type(self))` is very odd. Might `type(super(VIDEOGRAIN, self))` be better? (I think that would do the same.

Reply:
I'd just followed the examples seen on the web. See also https://bugs.python.org/issue14965 for reference. I added a comment that may still not be fully explanatory.

I tried `type(super(VIDEOGRAIN, self))` and the error reported was: `AttributeError: type object 'super' has no attribute 'length'`.

I verified that the existing call results in an `AttributeError` thrown in GRAIN's length setter as expected.